create_aio_app/template/{{cookiecutter.project_name}}/Makefile

```diff
@@ -28,6 +28,10 @@ clean:
 bash:
 	@docker exec -it $(PROJECT_NAME) bash
 
+## Runs script for upgrade dependencies:
+upgrade:
+	python3 -m $(PROJECT_NAME).utils.check-requirements.py
```

Comment:
looks much better 👍

Reply:
btw, we talked about running this command out from the docker to upgrade packages that related with a current project.
U can do that some like that:

*version 1*

`docker-compose.yml`
```yaml
upgrade:
    <<: *app
    container_name: {{ cookiecutter.project_name }}_upgrande
    command: python3 -m  {{ cookiecutter.project_name }}.utils.check-requirements.py
```
`Makefile`
```
upgrade:
 	docker-compose --rm run upgrade
```

*version 2*

or use only `Makefile`

`Makefile`
```
upgrade:
 	docker-compose --rm run app python3 -m $(PROJECT_NAME).utils.check-requirements.py
```

Personally, i prefer *version 2*, because we change only one file. Important part here is `--rm` don't forget to add that, please.

[docker-compose run - docs](https://docs.docker.com/compose/reference/run/)